### DISCUSSION TITLE
After the first hover, page and tooltip flickers

### DISCUSSION THREAD

**Discussion by ECBSJ** [ReactTooltip/react-tooltip#943] [Q&A]
On the first initial hover of the element after setting up tooltips, the tooltip shows and hides smoothly. 

But the next time I hover, and every hover after the first, the tooltip and webpage flickers.

Any suggestions?

I am using the latest version of `react-tooltip` in a webpack react app. I've also overwritten some of the default css options but only for `width`, `font-size`, and `text-align`.

**Comment by gabrieljablonski** [comment]
Can you provide a small sample project reproducing this problem?

**Comment by ECBSJ** [comment]
It could've been an issue with the tooltip clashing with my CSS style and setup. I'll close this discussion for now.
